logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt

```diff
@@ -115,6 +116,32 @@ class MLSConversationDataSource(
             }
         }
 
+    override suspend fun removeMembersFromMLSGroup(
+        clientId: ClientId,
+        groupID: String,
+        userIdList: List<UserId>
+    ): Either<CoreFailure, Unit> =
+        // TODO: check for federated and non-federated members
+        mlsClientProvider.getMLSClient().flatMap { client ->
+            val usersCryptoQualifiedClientIDs = userIdList
+                .map { CryptoQualifiedClientId(clientId.value, idMapper.toCryptoQualifiedIDId(it)) }
```

Comment:
You are creating a `CryptoQualifiedClientId` with a another user's `UserId` and the self user `ClientId`, that doesn't make sense. We need fetch the clientIDs for all the user's devices.

Reply:
Oh! I missed this point. how can I get all the others clientIds?

Reply:
found it: https://staging-nginz-https.zinfra.io/api/swagger-ui/#/default/post_users_list_clients correct?

Reply:
Yes, that one or https://staging-nginz-https.zinfra.io/api/swagger-ui/#/default/get_users__uid_domain___uid__clients